src/utils/calldata/enum/CairoCustomEnum.ts

```diff
@@ -45,9 +43,8 @@ export class CairoCustomEnum {
    * @returns the content of the valid variant of a Cairo custom Enum.
    */
   public unwrap(): any {
-    const variants = Object.entries(this.variant);
-    const activeVariant = variants.find((item) => !isUndefined(item[1]));
-    return isUndefined(activeVariant) ? undefined : activeVariant[1];
+    const variants = Object.values(this.variant);
+    return variants.find((item) => !isUndefined(item));
```

Comment:
Is this same expected case @PhilippeR26

Reply:
Well,
The current code is not my original code.
The initial intend was to return the content of the only one key that has a content defined, and if all are undefined (a case that should normally not occur) it returns undefined.
I find that ultra compact code is elegant, but is not easy to read!